LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Mac/MACCommandJsonConverter.cs

```diff
@@ -90,6 +86,9 @@ void GetValue(string propertyName, out JToken value)
                         serializer.Populate(item.CreateReader(), cmd);
                         return cmd;
                     }
+                    case Cid.Zero:
+                    case Cid.One:
```

Comment:
0 and 1 are invalid CIDs. We should consider removing these from `Cid` entirely.

Reply:
I had to keep `Cid.Zero` due to CA1008, but I renamed it to `None` to make it clearer where it comes from. Addressed with b09fe9b16edb8b8ce78be4b8cb6e618b41fa81de

Reply:
Since we are using the CID values from the spec directly as the enumeration member values, `Cid.None` doesn't bring much value (in fact, it has the downside that we can't rely on `Enum.IsDefined` to valid the CID). I would opt to disable CA1008 as we've done in other cases like `Bandwidth`, `ReceiveWindowNumber`, `SpreadingFactor`, etc.

Reply:
Addressed this with 76af7d871d2a8d99577995ee69b2615398b0835f. This caught one instance where we logged a wrong CID, this is now also fixed as part of the same commit.